src/BuildingRegistry.Api.BackOffice.Abstractions/BuildingUnit/Requests/RetireBuildingUnitRequest.cs

```diff
@@ -0,0 +1,27 @@
+namespace BuildingRegistry.Api.BackOffice.Abstractions.BuildingUnit.Requests
+{
+    using System.Collections.Generic;
+    using Be.Vlaanderen.Basisregisters.GrAr.Provenance;
+    using Be.Vlaanderen.Basisregisters.Sqs.Responses;
+    using BuildingRegistry.Building;
+    using BuildingRegistry.Building.Commands;
+    using MediatR;
+    using Newtonsoft.Json;
+
+    public class RetireBuildingUnitRequest : RetireBuildingUnitBackOfficeRequest, IRequest<ETagResponse>
+    {
+        [JsonIgnore]
+        public IDictionary<string, object> Metadata { get; set; }
+
+        public RetireBuildingUnit ToCommand(
+            BuildingPersistentLocalId buildingPersistentLocalId,
+            BuildingUnitPersistentLocalId buildingUnitPersistentLocalId,
```

Comment:
I think we can drop the BuildingUnitPersistentLocalId parameter and get its value from the BackOfficeRequest property?